ivy_tests/test_ivy/test_functional/test_core/test_manipulation.py

```diff
@@ -708,3 +713,30 @@ def test_unstack(
         axis=axis,
         keepdims=keepdims,
     )
+
+
+# gather_nd
+@handle_test(
+    fn_tree="paddle.gather_nd",
+    dtype_param_and_indices=_gather_helper(),
+)
+def test_paddle_gather_nd(
```

Comment:
the tests are failing on the ci, can you run this locally to ensure it passes? Thanks

Reply:
Hi @vaithak, sorry I just see your comment now. I will work on it and let you know. Thanks.

Reply:
![image](https://github.com/unifyai/ivy/assets/56084688/1f6947b5-293d-42c2-b58d-d768a5290e1e)

Update: Above is this output when running test on gather_nd. I am looking more into the error to fix it.